Chatto/Tests/ObservableAsyncStreamTests.swift

```diff
@@ -30,28 +30,28 @@ final class ObservableAsyncStreamTests: XCTestCase {
 
     func test_GivenStreamFromObservable_WhenObservableUpdates_ThenValueDeliveredToStream() async throws {
         let observable = Observable(0)
-        let stream = observable.asyncStream()
+        let stream = observable.values
+
         async let firstReceivedValue: Int? = stream.first { _ in true }
         observable.value = 1
-        let result = await firstReceivedValue
-        XCTAssertEqual(result, 1)
+        let first = await firstReceivedValue
+        XCTAssertEqual(first, 1)
+
+        async let secondReceivedValue: Int? = stream.first { _ in true }
+        observable.value = 2
+        let second = await secondReceivedValue
+        XCTAssertEqual(second, 2)
     }
 
-    func test_WhenObserverDeallocated_ThenStreamCompletes() async throws {
+    func test_WhenObservableDeallocated_ThenStreamCompletes() async throws {
         var observable: Observable<Void>? = .init(())
-        let stream = observable!.asyncStream()
+        let stream = observable!.values
         let iterateAllStreamValuesTask = Task {
             for await _ in stream {}
         }
         observable = nil
 
-        try await withTimeout {
-            await withTaskCancellationHandler {
-                await iterateAllStreamValuesTask.value
-            } onCancel: {
-                iterateAllStreamValuesTask.cancel()
-            }
-        }
+        try await withTimeout { await iterateAllStreamValuesTask.value }
```

Comment:
My personal opinion: I think it's still beneficial to see the Task being explicitly cancelled, rather than with a time-out.

That way we can also omit the timeout period.